blog/2021-08-14-react-navigation-6.0.md

```diff
@@ -0,0 +1,159 @@
+
+### New `Group` component for organization
+
+We have now added a [`Group`](/docs/group) component to organize your screens. You can group your screens using a `Group` component in any navigator. You can even nest `Group` components inside other `Group` components. They are lightweight and don’t render anything - like fragments, so they won’t affect performance.
+
+A nice feature of the `Group` component is that you can specify `screenOptions` on it like you can on a navigator. Passing `screenOptions` to a group configures all the screens inside that group to use these options. You can override these options by passing `options` to each Screen component, similar to how you can with `screenOptions` on Navigator.
+
+In this code snippet, you can see that we group regular screens under one group and modal screens under another group:
+
+```js
+function App() {
+  return (
+    <Stack.Navigator>
+      <Stack.Group>
+        <Stack.Screen name="Home" component={HomeScreen} />
+        <Stack.Screen name="Details" component={DetailsScreen} />
+      </Stack.Group>
+      <Stack.Group screenOptions={{ presentation: 'modal' }}>
+        <Stack.Screen
+          name="CreatePost"
+          component={CreatePostScreen}
+        />
+      </Stack.Group>
+    </Stack.Navigator>
+  );
+}
+```
+
+### Headers by default in Bottom Tabs & Drawer
+
+We often needed to show a header in the screens of drawer and bottom tabs. To do this, we had to nest a stack navigator which provides a header, even if it was a navigator with just one screen. So we now show headers by default in screens of drawer and bottom tabs. No nesting necessary.
+
+We also export a [`Header`](/docs/elements#header) component in the new elements library to use anywhere in your components.
+
+### Native navigation by default
+
+Historically, React Navigation has been mostly JS based, with animations and gestures written in JavaScript. While this works for a lot of apps, apps with heavy screens can suffer from poor performance. So we also wanted to address this by using native primitives for navigation.
+
+With React Navigation 5, we introduced [`@react-navigation/native-stack`](/docs/native-stack-navigator) package powered by [`react-native-screens`](https://github.com/software-mansion/react-native-screens), as well as a native backend for [`@react-navigation/material-top-tabs`](/docs/material-top-tab-navigator) powered by [`react-native-pager-view`](https://github.com/callstack/react-native-pager-view).
+
+In React Navigation 6, we made `@react-navigation/native-stack` the default choice for setting up Stack navigation. It uses UINavigationController on iOS and Fragments on Android to implement navigation natively. We also focused a lot on aligning the API of `@react-navigation/native-stack` with `@react-navigation/stack` so that it’d be easier to switch between them.
```

Comment:
```suggestion
In React Navigation 6, we made `@react-navigation/native-stack` the default choice for setting up Stack navigation. It uses `UINavigationController` on iOS and Fragments on Android to implement navigation natively. We also focused a lot on aligning the API of `@react-navigation/native-stack` with `@react-navigation/stack` so that it’ll be easier to switch between them.
```